lib/http.js

```diff
@@ -5,14 +5,18 @@ var Q = require('q');
 var AuthApiError = require('./errors/AuthApiError');
 var config = require('./config');
 
-function httpRequest(sdk, url, method, args, dontSaveResponse) {
+function httpRequest(sdk, url, method, args, dontSaveResponse, accessToken) {
```

Comment:
question: taking an parameter (type `Object`) seems slightly better? (it probably need to refactor many places)